package.json

```diff
@@ -62,8 +62,11 @@
     "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
     "eslint": "^8.8.0",
+    "next": "^12.x",
```

Comment:
There is one thing that i do not clear understand, but looks like PNPM doesn't install peer dependencies

Reply:
NPM ditched peer deps install since version 3, but then reintroduced it in version 7, I guess PNPM just didn't catch up :)